torch/_dynamo/eval_frame.py

```diff
@@ -575,9 +577,11 @@ def dynamo_normalization_capturing_compiler(
         gm: torch.fx.GraphModule, example_inputs
     ):
         nonlocal graph
+        nonlocal fakified_example_inputs
 
         assert graph is None, "whole graph export entails exactly one graph"
         graph = gm
+        fakified_example_inputs = example_inputs
```

Comment:
These are compile time inputs, if you want to extract them, please call them that.

Reply:
renamed to compile_time_inputs